### ISSUE TITLE
config command  doesn't exists

### ISSUE THREAD

**Issue by marcotradenet** [commercionetwork/commercionetwork#325]
## Summary of Bug

I tried to use `config` command, but doesn't exists.

## Version

v3.0.0

## Steps to Reproduce

`commercionetworkd config -h`

returns

```
Error: unknown command "config" for "commercionetworkd"
Run 'commercionetworkd --help' for usage.
```

**Comment by marcotradenet** [comment]
This issue will be fixed in version 7.0.1

**Comment by marcotradenet** [comment]
Fixed in #327
